lib/std/crypto.zig

```diff
@@ -119,8 +123,14 @@ pub const sign = struct {
 pub const stream = struct {
     pub const chacha = struct {
         pub const ChaCha20IETF = @import("crypto/chacha20.zig").ChaCha20IETF;
+        pub const ChaCha12IETF = @import("crypto/chacha20.zig").ChaCha12IETF;
+        pub const ChaCha8IETF = @import("crypto/chacha20.zig").ChaCha8IETF;
```

Comment:
remark / question

Interesting how these are not called `ChaCha20With96BitNonce`, `ChaCha12With96BitNonce` and `ChaCha8With96BitNonce` to match `ChaCha20With64BitNonce`, `ChaCha12With64BitNonce`, `ChaCha8With64BitNonce` respectively.